changes/13086-fix-text-cell-truncation

```diff
@@ -0,0 +1 @@
+- Text cells truncate with ellipses if longer than column width
```

Comment:
Wrong issue

Reply:
Nice catch! 🦅 👁️

Reply:
@jacobshandling 

I left a note in my PR that I forgot to add a change file for a previous issue, can you approve with the change file?

<img width="766" alt="Screenshot 2023-08-11 at 9 22 50 AM" src="https://github.com/fleetdm/fleet/assets/71795832/9ecaa812-e5d3-4f99-afc7-e71caf880cb8">